corehq/apps/hqwebapp/utils/bootstrap/paths.py

```diff
@@ -13,13 +13,40 @@
     "ex-submodules": COREHQ_BASE_DIR / "ex-submodules",
 }
 GRUNTFILE_PATH = COREHQ_BASE_DIR.parent / "Gruntfile.js"
+IGNORED_PATHS_BY_APP = {
+    "hqwebapp": [
+        "hqwebapp/js/resource_versions.js",
+        "hqwebapp/base.html",
+        "hqwebapp/base_mobile.html",
+        "hqwebapp/crispy/",
+        "hqwebapp/includes/inactivity_modal_data.html",
+        "hqwebapp/includes/core_libraries.html",
+        "hqwebapp/includes/ui_element_js.html",
+        "hqwebapp/partials/requirejs.html",
+        # todo, remove these files and fully split:
+        "hqwebapp/iframe_domain_login.html",
+        "hqwebapp/bulk_upload.html",
+        "hqwebapp/crispy/single_crispy_form.html",
```

Comment:
This is redundant since `"hqwebapp/crispy/"` is present above, right?

Reply:
that was there as a placeholder/reminder to migrate, but it looks like i can't do a general `/crispy/` path afterall

Reply:
turns out this was a bad idea to put `hqwebapp/crispy/` in there! Thanks for catching! would have missed a few more files :)